### DISCUSSION TITLE
Puzzle 19 Back to the Future

### DISCUSSION THREAD

**Discussion by maximilian22x** [smpetersgithub/AdvancedSQLPuzzles#17] [General]
https://github.com/smpetersgithub/AdvancedSQLPuzzles/blob/66533d023b70ef3d234a302a2453951bee29936f/Advanced%20SQL%20Puzzles/Advanced%20SQL%20Puzzles%20Solutions.sql#L851

Very confusing written assigment. Why **2018-01-10	2018-01-11** are in the results set? This time periode has no overlap. 
[overlapp.xlsx](https://github.com/smpetersgithub/AdvancedSQLPuzzles/files/10081186/overlapp.xlsx)

**Comment by smpetersgithub** [comment]
The question could be reworded to something such as....

Given the following records, create a result set that merges the overlapping time frames where applicable.  If a time frame does not overlap, include it in the result set as well.

**Comment by smpetersgithub** [comment]
Closing
